### DISCUSSION TITLE
Future support of MacOS - what versions will installation support?

### DISCUSSION THREAD

**Discussion by itoldusoandso** [input-leap/input-leap#1630] [Q&A] [answered]
I understand Input Leap install for MacOS has not been released yet. Is there some hint about what versions of MacOS will the installation file support? I am not expecting much in terms of active support for older MacOS (e.g. 10.12, 10.13, 10.14 ..), that's the least I would expect from a volunteer driven effort, but I hope the installation will support these MacOS versions.

My issue with current is the dreadful Error 65541 and also the inability to use SSL (not sure workarounds would work for me). So my biggest wish input-leap would not have these 2 issues.

PS: Any idea if the release will still happen this year, or shall I check again next year?

**Comment by Cimbali** [comment, marked as answer]
Currently CI builds use `CMAKE_OSX_DEPLOYMENT_TARGET=10.9` which [is translated to](https://cmake.org/cmake/help/latest/variable/CMAKE_OSX_DEPLOYMENT_TARGET.html) the `-mmacosx-version-min` flag, which is defined as the ”earliest version of MacOS X that this executable will run on”.

**Reply by shymega** [reply]
Thanks for this. I'll move the discussion to Q&A, and mark your answer as the correct one.

**Reply by Cimbali** [reply]
Note that I since then noticed that the `.dmg` includes libraries built for the platform where the package is bundled. This means likely macOS 11 would be oldest supported in that format -- but building for older targets is still tested for and supported by CIs.
